Documentation/getting-started.md

```diff
@@ -0,0 +1,121 @@
+# Adobe Campaign Classic
+
+## Configure Campaign Classic extension in the Data Collection UI
+
+1. In the Data Collection UI, click the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Campaign Classic** extension, and click **Install**.
+3. Type in the settings for your extension.
+4. Click **Save**.
+5. Complete the publishing process to update the SDK configuration.
+
+   For more information about publishing, see the [publishing overview](https://experienceleague.adobe.com/docs/launch/using/publish/overview.html).
+
+> ℹ️ You can retrieve your Campaign Classic registration or tracking endpoint URLs in the Campaign Classic interface under the **Tools > Advanced > Deployment wizard** menu. The endpoint for push notifications is usually the same as the URL that is used for web forms and surveys.
+
+#### Registration endpoints
+
+Type the registration endpoint URL(s) for your Campaign Classic instances. You can specify up to three unique endpoints for your development, staging, and production environments.
+
+> ⚠️ For this extension, the registration endpoint URLs should be entered **without** a prefixing `https://.`
+
+#### Tracking endpoints
+
+Type the tracking endpoint URL(s) for your Campaign Classic instances. Like the registration URLs, you can specify up to three unique endpoints for your development, staging, and production environments.
+
+> ⚠️ For this extension, the tracking endpoint URLs should be entered **without** a prefixing `https://.`
+
+#### Integration key (Android)
+
+You can specify up to three unique Android integration keys for your development, staging, and production environments. Android integration keys are generated after creating a service that contains Android applications using the [Campaign Classic client console](https://experienceleague.adobe.com/docs/campaign-classic/using/installing-campaign-classic/connect-to-campaign/installing-the-client-console.html). For more information on where to find the integration key, see the tutorial on [configuring the mobile application in Adobe Campaign](https://experienceleague.adobe.com/docs/campaign-classic/using/sending-messages/sending-push-notifications/configure-the-mobile-app/configuring-the-mobile-application-android.html).
+
+#### Request timeout
+
+The request timeout is the amount of time, in seconds, to wait for a response from the registration or tracking endpoint before timing out. The SDK default timeout value is 30 seconds.
+
+## Add Campaign Classic to your app
+
+1. Add the Campaign Classic extension to your project using the app's Gradle file.
+
+   ```gradle
+   implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+   implementation 'com.adobe.marketing.mobile:campaignclassic:1.+'
```

Comment:
these need to be updated to their respective 2.+ versions.  also remember our "core" extensions aren't all bundled in `sdk-core` any more